lib/addons/p5.dom.js

```diff
@@ -188,7 +188,10 @@
       return converted;
     } else if (elt.tagName === "VIDEO" || elt.tagName === "AUDIO") {
       return new p5.MediaElement(elt);
-    } else {
+    } else if ( elt.tagName === "SELECT" ){
+      return createSelect( new p5.Element(elt) );
+    } 
```

Comment:
If the tag name is select here, the addElement will never invoked insides createSelect. Because we're not going to add a new select.